src/app/common/header/header.tpl.html

```diff
@@ -10,14 +10,22 @@
   </div>
   <!-- Collect the nav links, forms, and other content for toggling -->
   <div class="collapse navbar-collapse" ng-class="!navCollapsed && 'in'" ng-show="!navCollapsed">
+    <div class="container" id="header-container">
+      <div class="row">
+        <div class="col-md-6">
     <ul class="nav navbar-nav navbar-left">
       <ul class="breadcrumb">
+        <div class="container" id="breadcrumb-container">
+          <div class="row">
+            <div class="col-md-1" id="breadcrumb-col1">
```

Comment:
Why does each column need a unique ID?

Reply:
I have given each column a unique id as I have given different padding for the columns.